tests/test_visitors/test_ast/test_naming/test_variable_self_reassignment.py

```diff
@@ -17,12 +19,34 @@
 test_variable = 10
 """
 
+wrong_fragment_double_assignment = """
+test_variable = 5
+test_variable = test_variable = 10
+"""
+
+wrong_fragment_other_assignment = """
+test_variable = 5
+test_variable = other = test_variable = 5
+"""
+
+wrong_fragment_tuple_assignment = """
+x = 1
+y = 2
+x, y = x, y
+"""
 
+
+@pytest.mark.parametrize('fragment', [wrong_fragment,
+                                      wrong_fragment_double_assignment,
+                                      wrong_fragment_other_assignment,
+                                      wrong_fragment_tuple_assignment,
+                                      ],
+                         )
 def test_self_variable_reassignment(
-    assert_errors, parse_ast_tree, default_options,
+    assert_errors, parse_ast_tree, fragment, default_options,
```

Comment:
We use `code` variable name for code samples, not `fragment`. Please, rename it.